scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
OK, well I asked 13 days ago if there was a reason not to use `--timeout` and noone mentioned that at the time. If you already tried it and it didn't work then I'm fine with this API hack, but we could have saved spending time on this discussion if someone had mentioned that earlier. I think we are all agreed that we want to get it working again ASAP.